### ISSUE TITLE
Test Azure Devops Integrasjon

### ISSUE THREAD

**Issue by okms** [Puzzlepart/prosjektportalen365#775]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Additional context**

**Is this something your organization is willing to contribute to financially?**
The Project Portal is a community-driven and dugnad-driven project, supported by organizations both in the public and private sector. If you're willing to contribute, please let us know (this will increase the chance of the feature being implemented). Note: This is not a commitment, we will get back to you with an estimate before starting to work on the issue.
